src/webgl/material.js

```diff
@@ -183,6 +183,65 @@ p5.prototype.createShader = function(vertSrc, fragSrc) {
   return new p5.Shader(this._renderer, vertSrc, fragSrc);
 };
 
+/**
+ * Like <a href="#/createShader">createShader()</a>, but with
+ * a default vertex shader so that only a fragment shader is supplied.
+ * Creates a new <a href="#/p5.Shader">p5.Shader</a> object.
+ *
+ * Note, shaders can only be used in WEBGL mode.
+ *
+ * @method createFilterShader
+ * @param {String} fragSrc source code for the fragment shader
+ * @returns {p5.Shader} a shader object created from the provided
+ *                      fragment shader.
+ * @example
+ * <div modernizr='webgl'>
+ * <code>
+ * function setup() {
+ *   let frag = `precision mediump float;
+ *   varying mediump vec2 vTexCoord;
+ *
+ *   uniform sampler2D tex0;
+ *
+ *   float luma(vec3 color) {
+ *     return dot(color, vec3(0.299, 0.587, 0.114));
+ *   }
+ *
+ *   void main() {
+ *     vec2 uv = vTexCoord;
+ *     // gets flipped vertically
+ *     uv.y = 1.0 - uv.y;
```

Comment:
Should we preflip the y uv so that the user doesn't need to deal with this?

Reply:
Just curious, would the user be able to understand that this is being done for them?

Reply:
Yeah, after trying the pre-flipped `vTexCoords.y` it for a week I came across passing mouseY as a uniform and forgetting to invert the coordinates. So the user will eventually find out that they have to flip y, and the inconsistency might be confusing between when it's done for them (maybe this one time out of convenience) and when it's not (the rest of the time).